src/xenia/memory.h

```diff
@@ -36,6 +36,13 @@ enum SystemHeapFlag : uint32_t {
   kSystemHeapDefault = kSystemHeapVirtual,
 };
 
+enum HeapTypeFlag : uint8_t {
```

Comment:
These are mutually exclusive, so should be just an `enum class`, not bits (and `HeapType` then, not `HeapTypeFlag`).